src/lib/validation.ts

```diff
@@ -1,28 +1,24 @@
 import * as _ from 'lodash';
 import { inspect } from 'util';
 
+import { EnvVarObject, LabelObject } from './types';
+
 export interface CheckIntOptions {
 	positive?: boolean;
 }
 
-export interface EnvVarObject {
-	[name: string]: string;
-}
-
-export interface LabelObject {
-	[name: string]: string;
-}
-
 const ENV_VAR_KEY_REGEX = /^[a-zA-Z_][a-zA-Z0-9_]*$/;
 const LABEL_NAME_REGEX = /^[a-zA-Z][a-zA-Z0-9\.\-]*$/;
 
+type NullableString = string | undefined | null;
```

Comment:
Should this also go in types.ts?

Reply:
I'm not sure, I've tried to stick to only moving types that will be used in another place, whereas this was simply to avoid having to write out the value at every place. If another module finds itself needing it, I'll definitely move it out.